support/templates/nvidia-device-plugin.yaml

```diff
@@ -0,0 +1,50 @@
+{{ if .Values.config.nvidiaDevicePlugin.enabled }}
```

Comment:
maybe nvidiaDevicePlugin.azure.enabled? This is using what looks like an Azure specific image.

Reply:
Done!

Reply:
I renamed the template file too, to be extra clear